src/app/core/components/generic-mod-list/generic-mod-list.component.scss

```diff
@@ -68,6 +74,10 @@ mat-card {
   flex-direction: row;
 }
 
+.card-button-container {
+  display: flex;
+  max-height: 44px;
```

Comment:
This works not 100% for the button because there are multiple sizes which should be changed. There is the button and ripple effect. This line can be removed if mat-card-content width will be changed


Just all width:100% to the mat-card-content line 17

Reply:
Removed max-height and with the smaller 'last update' text, no other changes were necessary.

Reply:
Sorry. Messed up. I meant to add HEIGHT of 100% to the mat-card-content. Sorry.
Apply the height will prevent the content to overlapping the container.

Without
![grafik](https://github.com/SchraderR/sp-tarkov-client/assets/49452108/4165b502-9e90-4da4-bc06-e47863dabc8f)

With
![grafik](https://github.com/SchraderR/sp-tarkov-client/assets/49452108/9239dc63-b9bd-4af7-90b4-7540f4e47052)

Reply:
Oh I see what you mean. I tried height also but at first I didn't see the difference.
Added height: 100% to mat-card-content